pallets/omnipool/src/lib.rs

```diff
@@ -178,14 +178,18 @@ pub mod pallet {
 	/// Total amount of hub asset reserve. It equals to sum of hub_reserve of each asset in omnipool
 	pub(super) type HubAssetLiquidity<T: Config> = StorageValue<_, T::Balance, ValueQuery>;
 
+	#[pallet::storage]
+	/// Tradable state of hub asset.
+	pub(super) type HubAssetTradability<T: Config> = StorageValue<_, Tradable, ValueQuery>;
```

Comment:
Optimization: I suspect these are mostly accessed together and could be combined?

Reply:
I was thinking the same. but i discarded it for some reason. Now i need to remember the reason 🤔 

let me check.

Reply:
I had a look again. I don't think there is a reason why it cannot be done.

I decided to this way - because the Tradable state of hub asset is only needed when you trade Hub asset - which is currently not allowed to buy, only to sell. And to obtain LRNA, you can only obtain hub asset after remove_liquidity and only if the price changes in specific way. 

But HubAssetLiquidity is used almost in all calls. so the state would be read alongside the total hub liquidity every time even it is not needed.

Although the trade state is just enum and probably would not make such difference in the weight, I think leaving these this piece of information of hub asset separated is fine. 

Is there anything else you can think of to consider ?

Reply:
HubAssetLiquidity has been removed.